website/docs/cards.md

```diff
@@ -123,6 +123,7 @@ Use stats cards for a quick and easy way to display live values and graphics (or
 | [ui.large_stat_card()](api/ui.md/#large_stat_card) | Stat card; displays a primary value, an auxiliary value and a caption. |
 | [ui.tall_gauge_stat_card()](api/ui.md/#tall_gauge_stat_card) | Tall stat card; displays a primary value, an auxiliary value and a progress gauge. |
 | [ui.tall_series_stat_card()](api/ui.md/#tall_series_stat_card) | Tall stat card; displays a primary value, an auxiliary value and a series plot. |
+| [ui.tall_stats_card()](api/ui.md/#tall_stats_card) | Tall stats card; display many stats vertically |
```

Comment:
I'm not sure how the documentation should be done and if the reference to the page is right.

Reply:
Heh, didn't even know about this one, you can add it here, but the description needs work.

Reply:
I reworded the description.